docs/developer/flow/engines/base/frame/data.rst

```diff
@@ -0,0 +1,33 @@
+BasePandasFrame
```

Comment:
Isn't we placing all of the flow docs in "flow/**modin**/..."? At least #2952 sets this pattern

Reply:
This needs to be discussed on today's meeting.

Reply:
https://github.com/modin-project/modin/tree/master/docs/flow/modin

Let's use this